### PR TITLE
mutant: add INVERT_LOOPCTRL mutation

### PR DESCRIPTION
**Pull request by alessio-perugini**
## Proposed changes

Implement #136 invert loop break/continue

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/go-gremlins/gremlins/docs/CONTRIBUTING.md) doc
- [x] Lint and unit tests pass locally with my changes (`make all`)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

## Further comments

I've added the loop control support. I'm not sure if the fixtures are good enough.
I was wondering what will happen when changing the control results in an infinite loop?

### CODE REVIEW

**Comment by k3rn31** [comment]
Hello @alessio-perugini, thanks for you PR. I enabled the CI for you.

**Comment by alessio-perugini** [comment]
@k3rn31 thank you. I run a simple test when the mutant generates an infinite loop and the result is a `TIMED OUT`. Are we okay with that?

**Comment by k3rn31** [comment]
@alessio-perugini I'd say it is to be expected in this kind of mutation. It is showing a brittle test case :)

As soon as you mark it as non draft we'll proceed with the review.

File: docs/docs/usage/mutations/invert_loop.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: Invert loop
+---
+
+# Invert loop control
+
+_Invert loop control_ will perform inversions on control operations, which means a `continue` will become a `break`..
```
Comment (k3rn31) [inline docs/docs/usage/mutations/invert_loop.md:7]: The sentence ends with two periods.

**Review by k3rn31 — CHANGES_REQUESTED** [review]
Thanks again for the PR. Everything looks good.
Since this kind of mutation can produce lots of unbound loops and, consequently, timeouts, it is maybe better to keep it disabled by default.
Could you please fix that and update the documentation accordingly?

Also, since every mutation increases testing times, we decided to keep all the new mutations disabled by default, leaving to the user the decision to enable them on the basis of project necessities. In the future we'll make some statistics to craft a better core set.

**Review by k3rn31 — APPROVED** [review]
Everything looks good, I'm going to merge this.

**Pull request merged**